scripts/exrrfs_make_ics.sh

```diff
@@ -956,6 +956,34 @@ In directory:    \"${scrfunc_dir}\"
 #
 #-----------------------------------------------------------------------
 #
+# Create a variable definitions file (a shell script) and save in it the
+# values of several external-model-associated variables generated in this
+# script that will be needed by downstream workflow tasks.
+#
+#-----------------------------------------------------------------------
+#
+extrn_mdl_var_defns_fn="${EXTRN_MDL_ICS_VAR_DEFNS_FN}"
+extrn_mdl_var_defns_fp="${extrn_mdl_staging_dir}/${extrn_mdl_var_defns_fn}"
+check_for_preexist_dir_file "${extrn_mdl_var_defns_fp}" "delete"
+
+settings="\
+EXTRN_MDL_CDATE=\"${extrn_mdl_cdate}\"
```

Comment:
969 and 970 need to change to:
settings="EXTRN_MDL_CDATE=${extrn_mdl_cdate}"

Reply:
Sure I will get that fixed now.  Having it on multiple lines was needed when there were multiple variables included in "settings".